src/Internal/Workflow/Process/Scope.php

```diff
@@ -322,16 +324,25 @@ function () use ($cancelID): void {
      */
     protected function call(callable $handler, ValuesInterface $values): \Generator
     {
-        $this->makeCurrent();
-        $result = $handler($values);
+        try {
+            $this->makeCurrent();
+            try {
+                $result = $handler($values);
+            } catch (InvalidArgumentException) {
+                // Skip deserialization errors
```

Comment:
Just want to confirm there aren't any other exception types that could result from bad deserialization that need to be included here.

Reply:
As you can see it throws SDK's `InvalidArgumentException` only
https://github.com/temporalio/sdk-php/blob/cf2f921a6ca923f7e6ccffc3d12314c445af236a/src/Internal/Declaration/Dispatcher/AutowiredPayloads.php#L30-L36